Formula/libui.rb

```diff
@@ -0,0 +1,50 @@
+class Libui < Formula
+  desc "portable GUI library for C"
+  homepage "https://github.com/andlabs/libui"
+  url "https://github.com/andlabs/libui/archive/alpha3.1.tar.gz"
+  sha256 "945ee9abacffafa93b5cf941bddfbe0bd0d4b3a8c7f8b0256779e24249f57679"
+
+  head "https://github.com/andlabs/libui/libui.git"
+
+  option "without-shared-library", "Build static library only (defaults to building dylib only)"
+  option "with-examples", "Build examples"
+  option "with-tester", "Build tester program"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = []
+    args << "-DBUILD_SHARED_LIBS=OFF" if build.without? "shared-library"
+    args << ".."
+
+    mkdir "build" do
+      system "cmake", *args
+      system "make"
+      system "make", "examples" if build.with? "examples"
+      system "make", "tester" if build.with? "tester"
+
```

Comment:
Do you know if this project's `cmake` generates a `make install` target we can use instead of manually calling `libexec`/`lib`/`include`? The most common `cmake` flow looks something like this, usually:

```ruby
args = std_cmake_args
args << "-DBUILD_SHARED_LIBS=OFF" if build.without? "shared-library"

mkdir "build" do
  system "cmake", "..", *args
  system "make"
  # additional targets
  system "make", "install"
end
```